spec/usage_spec.lua

```diff
@@ -169,10 +169,38 @@ describe('usage', function()
     end)
   end)
 
-  it("encode_format", function()
-    local usage = Usage.new()
-    usage:add('hits', 0)
-    assert.are.same(usage:encoded_format(), "usage%5Bhits%5D=0")
+  describe("encode format", function()
+    it("works with a single metric", function()
+      local usage = Usage.new()
+      usage:add('hits', 0)
+      assert.are.same(usage:encoded_format(), "usage%5Bhits%5D=0")
+    end)
+
+    it("with multiple metrics return the same", function()
+      local usage = Usage.new()
+      usage:add('a', 0)
+      usage:add('c', 0)
+      usage:add('b', 0)
+      assert.are.same(usage:encoded_format(), "usage%5Ba%5D=0&usage%5Bb%5D=0&usage%5Bc%5D=0")
+      -- Multiple cases just in case
+      local usage = Usage.new()
+      usage:add('c', 0)
+      usage:add('a', 0)
+      usage:add('b', 0)
+      assert.are.same(usage:encoded_format(), "usage%5Ba%5D=0&usage%5Bb%5D=0&usage%5Bc%5D=0")
+    end)
+
+    it("integer metrics", function()
```

Comment:
Do we need to test this? I don't think this is supported :thinking:

Reply:
I was thinking the same, but didn't harm to have just in case.